ebb_lint/flake8.py

```diff
@@ -273,6 +273,7 @@ def _check_tree(self, tree):
                 if not pattern.match(node, results):
                     continue
                 for k in extra.get('comments_for', ()):
+                    # XXX: this doesn't use `k` for finding the node
```

Comment:
What does it mean to use the `k` in `for k in extra.get('comments_for', ()):` to find a node?